conf/application.conf

```diff
@@ -238,6 +238,10 @@ voa {
       detailedValuation.path = "/external-valuation-management-api/properties/{uarn}/valuations/{valuationId}"
       valuationLookupUrl.path = "/external-valuation-management-api/valuationLookups/{lookupType}"
     }
+
+    customerManagementApi {
+      getAgentByRepresentationCode.path = "/customer-management-api/organisation"
```

Comment:
this url is a generic URL we may wish to use in other ways, this is get my organisation endpoint.

Reply:
will rename to getOrganisationByAgentCode